__DEFINES/map.dm

```diff
@@ -0,0 +1,40 @@
+///Map generation defines
+#define BIOME_LOWEST_HUMIDITY "biome_lowest_humidity"
+#define BIOME_LOW_HUMIDITY "biome_low_humidity"
+#define BIOME_MEDIUM_HUMIDITY "biome_medium_humidity"
+#define BIOME_HIGH_HUMIDITY "biome_high_humidity"
+#define BIOME_HIGHEST_HUMIDITY "biome_highest_humidity"
+
+#define BIOME_COLDEST "coldest"
+#define BIOME_COLD "cold"
+#define BIOME_WARM "warm"
+#define BIOME_TEMPERATE "perfect"
+#define BIOME_HOT "hot"
+#define BIOME_HOTTEST "hottest"
+
+#define BIOME_COLDEST_CAVE "coldest_cave"
+#define BIOME_COLD_CAVE "cold_cave"
+#define BIOME_WARM_CAVE "warm_cave"
+#define BIOME_HOT_CAVE "hot_cave"
+
+#define CLIMATE_ARCTIC	/datum/climate/arctic
```

Comment:
Some years ago I was asked by Damian why hide the fact that I was using typepaths with defines on a PR i can't remember. It was a very good question to which I didn't have an answer. I'm asking the same here.

Why are you using defines instead of just the typepath? In no context (`initial()`, `istype()`...) is using a typepath better.